.golangci.yml

```diff
@@ -104,4 +104,8 @@ linters:
     - ifshort
     - nosnakecase
     - depguard # Dependency whitelist, needs to be configured
+    - goconst # Fails if the same strings used in several places but is not a constant. Not very useful.
+    - inamedparam # Fails if interfaces do not have named params. Not in our code style.
+    - perfsprint # Suggests using strconv.* instead of fmt.* for printing numbers. Not very practical.
+    - protogetter # Complains when code reads var.Something if var.GetSomething() exists. Not useful.
```

Comment:
@pablochacin I've disabled these new linters as I personally did not find them very useful. LMK your thoughts about it, if you think they can be useful we can re-enable them and make the changes they suggest to the code.

Reply:
I reenabled goconst, it was not too hard to please it. I de-splitted some `t.Errorf` that were causing it to fire, and added a nolint directive to an iptables rule fragment we use quite a lot.